files/help/ares-device.help

```diff
@@ -6,16 +6,17 @@
     "synopsis" : {
         "default" : [
             "ares-device",
-            "ares-device [OPTION...]",
-            "ares-device --resource-monitor <APP_ID || SERVICE_ID>"
+            "ares-device [OPTION...]"
         ]
     },
     "description" : {
         "default": [
             "This command displays the device information and supports screen capture.",
             "",
-            "APP_ID is ID of the specific app for display its resource usage.",
-            "SERVICE_ID ID of the specific service for display its resource usage.",
+            "ID is the specific app or service for display its resource usage.",
+            "",
+            "FILE_PATH is the output directory or file path to save resource usage.",
```

Comment:
I think it is better to delete "output directory or"

Reply:
CSV_FILE and CAPTURE_FILE
file name or file path.

Reply:
Fixed